src/main/java/com/redhat/devtools/intellij/quarkus/QuarkusConstants.java

```diff
@@ -12,6 +12,7 @@
 
 import com.intellij.openapi.util.Key;
 import com.redhat.devtools.intellij.quarkus.module.QuarkusExtensionsModel;
+import com.redhat.devtools.intellij.quarkus.module.QuarkusModel;
```

Comment:
Is all class from `com.redhat.devtools.intellij.quarkus.module` are used for wizard? If it that I find it is hard to understand this package is for wizard, I suggest that you rename to `com.redhat.devtools.intellij.quarkus.projectWizard` like IJ provides com.intellij.ide.util.`projectWizard`.*

Reply:
done